products/pycharm.md

```diff
@@ -1,19 +1,33 @@
 ---
-title: PyCharm
+title: PyCharm Plugin Development
 ---
 
-## Which API should be used by PyCharm plugin developers?
+## Introduction
+PyCharm is an IntelliJ Platform-based product.
+Plugin projects for PyCharm can be developed using IntelliJ IDEA with the Gradle plugin.
 
-### Platform Open API
+## Configuring Plugin Projects Targeting PyCharm
+The configuration of PyCharm plugin projects follows the methods described in [Configuring Plugin Projects using a Product-Specific Attribute](dev_alternate_products.md#configuring-plugin-projects-using-a-product-specific-attribute), and [Configuring the plugin.xml File](dev_alternate_products.md#configuring-the-plugin-pluginxml-file).
+The table below summarizes the attributes to set in the `build.gradle` file:
 
-The APIs from the standard set of Platform Open API modules can be safely used in plugins. Public symbols defined in these modules have a lifecycle, and will be marked as deprecated for several releases before being removed.
+| Gradle Plugin Attribute | Value |
+|-----------|-------|
+| `intellij.type` | `PY` or `PC` |
```

Comment:
please explain what PY and PC stand for

Reply:
Per your comment on IJSDK-648A, I changed "Gradle plugin" to "gradle-intellij-plugin".

I added links from the entries in the "`gradle-intellij-plugin` Attributes" column to the relevant portions of the gradle-intellij-plugin README for each attribute. Seems handy.

I also added "`PY` for PyCharm Professional Edition, or `PC` for PyCharm Community Edition"